src/types/bitcoinTypes.ts

```diff
@@ -1 +1,22 @@
+import { Bytes } from "@polkadot/types";
+
 export type BitcoinNetwork = "mainnet" | "testnet" | "regtest";
+export type TxInclusionDetails = {
+    merkleProof: Bytes;
+    rawTx: Bytes;
+}
+export type TxInclusionDeterminants = {
+    btcTxId: string,
+}
+export type TxFetchingDetails = TxInclusionDetails | TxInclusionDeterminants;
```

Comment:
Not sure about the naming of some of these types, but can't really think of an alternative. At the very least `TxInclusionDeterminants` probably doesn't need to be a complex type.